iOSTest/UI/AuthorityList/AuthorityListView.swift

```diff
@@ -19,16 +20,37 @@ struct AuthorityListView: View {
                         }
                     }
                     .navigationTitle("Local Authorities")
-                    .navigationBarTitleDisplayMode(.inline)
+                    .toolbar {
+                        Button {
+                            showingCopyright.toggle()
+                        } label: {
+                            Image(systemName: "info.circle")
+                        }
+                    }
+                    .alert("Copyright © 2022 Infinity Works, Part of Accenture.", isPresented: $showingCopyright) {
+                    } message: {
+                        Text("All rights reserved.")
+                    }
```

Comment:
The new launch screen loses the copyright message, so I added it as a toolbar button alert instead.